app/components/windows/EventFilterMenu.tsx

```diff
@@ -0,0 +1,163 @@
+                  filter === 'resub' && (
+                    <div>
+                      <BoolInput
+                        value={this.resubFilters[filter].value}
+                        metadata={{ title: $t(this.resubFilters[filter].name) }}
+                        onInput={(value: boolean) => this.updateFilter(filter, value)}
+                        class={styles.categoryHeader}
+                      />
+                    </div>
+                  ),
+              )}
+              <div class={styles.resubOptions}>
+                {Object.keys(this.resubFilters).map(
+                  filter =>
+                    filter !== 'resub' &&
+                    this.resubsEnabled &&
+                    filter !== 'filter_subscription_minimum_months' &&
+                    filter !== 'filter_subscription_minimum_enabled' && (
+                      <div>
+                        <BoolInput
+                          value={this.resubFilters[filter].value}
+                          metadata={{ title: $t(this.resubFilters[filter].name) }}
+                          onInput={(value: boolean) => this.updateFilter(filter, value)}
+                        />
+                      </div>
+                    ),
+                )}
+              </div>
+              <div class={styles.minimum}>
+                {Object.keys(this.resubFilters).map(
+                  filter =>
+                    filter === 'filter_subscription_minimum_enabled' &&
+                    this.resubsEnabled && (
+                      <div>
+                        <BoolInput
+                          value={this.resubFilters[filter].value}
+                          metadata={{ title: $t(this.resubFilters[filter].name) }}
+                          onInput={(value: boolean) => this.updateFilter(filter, value)}
+                        />
+                      </div>
+                    ),
+                )}
+                {Object.keys(this.resubFilters).map(
+                  filter =>
+                    filter === 'filter_subscription_minimum_months' &&
+                    this.resubsEnabled && (
+                      <div class={styles.monthsInputContainer}>
+                        <NumberInput
+                          value={this.resubFilters[filter].value}
+                          metadata={{ min: 1, max: 120, isInteger: true }}
+                          onInput={(value: number) => this.updateFilter(filter, value)}
+                          class={styles.monthsInput}
+                        />
+                        <p class={styles.monthsLabel}>{this.resubFilters[filter].name}</p>
+                      </div>
+                    ),
+                )}
+              </div>
+            </div>
+          </div>
+        </div>
+        <div slot="controls">
+          <button class="button button--action" onClick={this.cancel}>
+            {$t('Done')}
+          </button>
+        </div>
+      </ModalLayout>
+    );
```

Comment:
Low priority but should probably split this up into some nicely named helper functions.